packages/flutter/lib/src/gestures/events.dart

```diff
@@ -420,7 +415,7 @@ abstract class PointerEvent extends Diagnosticable {
   /// Set if the event was synthesized by Flutter.
   ///
   /// We occasionally synthesize PointerEvents that aren't exact translations
-  /// of [ui.PointerData] from the engine to cover small cross-OS discrepancies
+  /// of [PointerData] from the engine to cover small cross-OS discrepancies
```

Comment:
Does this not have to be `[dart:ui.PointerData]`?

Reply:
I think either one would work - this works because we're importing it in this file.